muikku-elastic-search/src/main/java/fi/otavanopisto/muikku/plugins/search/ElasticSearchProvider.java

```diff
@@ -130,12 +132,28 @@ public SearchResult searchUsers(String text, String[] textFields, Collection<Env
         }
       }
       
+      if (excludeSchoolDataIdentifiers != null) {
+        IdsFilterBuilder excludeIdsFilterBuilder = FilterBuilders.idsFilter("User");
+        for (SchoolDataIdentifier excludeSchoolDataIdentifier : excludeSchoolDataIdentifiers) {
+          excludeIdsFilterBuilder.addIds(String.format("%s/%s", excludeSchoolDataIdentifier.getIdentifier(), excludeSchoolDataIdentifier.getDataSource()));
```

Comment:
Ids are (for some reason?) indexed like this, there is method toString in SchoolDataIdentifier but it returns a string where identifier and datasource are in wrong order.

Similar method was used in already existing "includeIds" filter:
```java
if (userIdentifiers != null) {
  IdsFilterBuilder idsFilter = FilterBuilders.idsFilter("User");
   for (SchoolDataIdentifier userIdentifier : userIdentifiers) {
     idsFilter.addIds(String.format("%s/%s", userIdentifier.getIdentifier(), userIdentifier.getDataSource()));
   }
   filters.add(idsFilter);
}

```